upload/catalog/model/catalog/product.php

```diff
@@ -47,7 +47,7 @@ public function getProduct($product_id) {
 				'height'           => $query->row['height'],
 				'length_class_id'  => $query->row['length_class_id'],
 				'subtract'         => $query->row['subtract'],
-				'rating'           => round($query->row['rating']),
+				'rating'           => $query->row['rating'] ? round($query->row['rating']) : 0,
```

Comment:
This should also work, isn't it?:

```  => round($query->row['rating'] ?? 0),```

Reply:
Yes but some people may be still on PHP5

Reply:
Oh, OK. If this is meant to be also back compatible that will not work